django_tasks/backends/database/models.py

```diff
@@ -66,6 +66,9 @@ class DBTaskResult(GenericBase[P, T], models.Model):
         max_length=max(len(value) for value in ResultStatus.values),
     )
 
+    enqueued_at = models.DateTimeField()
```

Comment:
Suggestion: You can use `auto_now_add=True` for this to let Django deal with making sure it's set during creation

Reply:
two lines less and future omissions prevented. Good point!